--- ./Makefile.in.org	Mon Aug 23 14:56:08 2004
+++ ./Makefile.in	Mon Aug 23 15:24:50 2004
@@ -44,7 +44,7 @@
 scriptdir=@scriptdir@
 utildir=@utildir@
 docdir=@docdir@
-vardir=@vardir@
+vardir=@localstatedir@
 tmpdir=@tmpdir@
 lockdir=@lockdir@
 
--- ./configure.in.org	Mon Aug 23 14:56:49 2004
+++ ./configure.in	Tue Aug 24 13:27:01 2004
@@ -1,5 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(bin/rip/whois_rip.c)
+AC_INIT([ripe-dbase], [3.2.0], [dbrip@ripe.net])
+AC_CONFIG_SRCDIR(bin/rip/whois_rip.c)
 AC_CONFIG_HEADER(include/config.h)
 
 AC_CONFIG_SUBDIRS(utils/whoisRIP)
@@ -23,21 +24,19 @@
 dnl installation directory layout
 dnl
 topdir=$prefix
-bindir=${topdir}/bin
-confdir=${topdir}/conf
-scriptdir=${topdir}/scripts
-utildir=${topdir}/utils
-docdir=${topdir}/doc
-vardir=${topdir}/var
-tmpdir=${vardir}/tmp
-lockdir=${vardir}/lock
+ripedata=${datadir}/ripe
+scriptdir=${ripedata}/scripts
+ripelib=${libdir}/ripe
+utildir=${ripelib}/utils
+docdir=${datadir}/doc
+tmpdir=${localstatedir}/tmp
+lockdir=${localstatedir}/lock
 
-libdir=${topdir}/lib/gnudl
 mandir=${topdir}/man/man1
 
 #######
 # Log infrastructure
-logdir=${topdir}/log
+logdir=${localstatedir}/log/ripe
 qrylog=${logdir}/qry
 dbuperr=${logdir}/dbuperr
 allriperr=${logdir}/riperr
@@ -53,12 +52,9 @@
 crosslog=${logdir}/cross
 
 AC_SUBST(topdir)
-AC_SUBST(bindir)
-AC_SUBST(confdir)
 AC_SUBST(scriptdir)
 AC_SUBST(utildir)
 AC_SUBST(docdir)
-AC_SUBST(vardir)
 AC_SUBST(tmpdir)
 AC_SUBST(lockdir)
 
@@ -78,17 +74,32 @@
 AC_SUBST(crosslog)
 
 AC_SUBST(docdir)
-AC_SUBST(libdir)
 AC_SUBST(mandir)
 
-
+#####################
+### Configs location:
+###
+dnl Where are configs...
+AC_ARG_WITH(config-dir,
+	    AC_HELP_STRING([--with-config-dir=DIR],[Set config file path (default: sysconfdir/conf)]),
+	    if test "x$with_config_dir" = xyes; then
+		    RIPE_CONFIGDIR="${sysconfdir}/conf"
+	    else
+		    if test "x$with_config_dir" = xno; then
+			    AC_MSG_ERROR(Config directory required)
+		    else
+			    RIPE_CONFIGDIR="$with_config_dir"
+		    fi
+	    fi,
+	    RIPE_CONFIGDIR="${sysconfdir}/conf")
+AC_SUBST(confdir, $RIPE_CONFIGDIR)
 
 dnl
 dnl glib-config
 dnl
 
 AC_ARG_WITH(glibconfig, 
-	[  --with-glibconfig=FILE     FILE=path of glib-config file[$ac_default_prefix/bin/glib-config]],
+	AC_HELP_STRING([--with-glibconfig=FILE],[FILE=path of glib-config file[$ac_default_prefix/bin/glib-config]]),
 	if test [ x$withval != x ] -a [ -x $withval ]; then
 	  glibconfig=$withval
 	else
@@ -108,7 +119,7 @@
 dnl
 
 AC_ARG_WITH(mysql, 
-	[  --with-mysql=DIR        DIR=path of MySQL distribution dir[$ac_default_prefix/mysql]],
+	AC_HELP_STRING([--with-mysql=DIR],[DIR=path of MySQL distribution dir[$ac_default_prefix/mysql]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  mysql=$withval
 	else
@@ -128,7 +139,7 @@
 dnl
 
 AC_ARG_WITH(mysqlinc, 
-	[  --with-mysqlinc=DIR        DIR=path of MySQL include dir[$ac_default_prefix/mysql/include]],
+	AC_HELP_STRING([--with-mysqlinc=DIR],[DIR=path of MySQL include dir[$ac_default_prefix/mysql/include]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  mysqlincdir=$withval
 	else
@@ -147,7 +158,7 @@
 dnl
 
 AC_ARG_WITH(mysqllib, 
-	[  --with-mysqllib=DIR        DIR=path of MySQL library dir[$ac_default_prefix/mysql/lib]],
+	AC_HELP_STRING([--with-mysqllib=DIR],[DIR=path of MySQL library dir[$ac_default_prefix/mysql/lib]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  mysqllibdir=$withval
 	else
@@ -166,7 +177,7 @@
 dnl
 
 AC_ARG_WITH(mysqlbin, 
-	[  --with-mysqlbin=DIR        DIR=path of MySQL binary dir[$ac_default_prefix/mysql/bin]],
+	AC_HELP_STRING([--with-mysqlbin=DIR],[DIR=path of MySQL binary dir[$ac_default_prefix/mysql/bin]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  mysqlbindir=$withval
 	else
@@ -185,7 +196,7 @@
 dnl
 
 AC_ARG_WITH(xsltconfig, 
-	[  --with-xsltconfig=FILE     FILE=path of xsltconfig file[$ac_default_prefix/bin/xslt-config]],
+	AC_HELP_STRING([--with-xsltconfig=FILE],[FILE=path of xsltconfig file[$ac_default_prefix/bin/xslt-config]]),
 	if test [ x$withval != x ] -a [ -x $withval ]; then
 	  xsltconfig=$withval
 	else
@@ -208,7 +219,7 @@
 dnl
 
 AC_ARG_WITH(xmlconfig, 
-	[  --with-xmlconfig=FILE     FILE=path of xml-config file[$ac_default_prefix/bin/xml2-config]],
+	AC_HELP_STRING([--with-xmlconfig=FILE],[FILE=path of xml-config file[$ac_default_prefix/bin/xml2-config]]),
 	if test [ x$withval != x ] -a [ -x $withval ]; then
 	  xmlconfig=$withval
 	else
@@ -231,7 +242,7 @@
 dnl
 
 AC_ARG_WITH(cclientinc, 
-	[  --with-cclientinc=DIR      DIR=path of c-client include dir[$ac_default_prefix/build/imap/c-client]],
+	AC_HELP_STRING([--with-cclientinc=DIR],[DIR=path of c-client include dir[$ac_default_prefix/build/imap/c-client]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  cclientinc=$withval
 	else
@@ -240,7 +251,7 @@
 	cclientinc=$ac_default_prefix/build/imap/c-client)
 
 AC_ARG_WITH(cclientlib, 
-	[  --with-cclientlib=DIR      DIR=path of c-client library dir[$ac_default_prefix/build/imap/c-client]],
+	AC_HELP_STRING([--with-cclientlib=DIR],[DIR=path of c-client library dir[$ac_default_prefix/build/imap/c-client]]),
 	if test [ x$withval != x ] -a [ -d $withval ]; then
 	  cclientlib=$withval
 	else
@@ -266,7 +277,7 @@
 dnl
 
 AC_ARG_WITH(gpgcmd, 
-	[  --with-gpgcmd=CMD      CMD=pathname of gpg executable [/usr/local/bin/gpg]],
+	AC_HELP_STRING([--with-gpgcmd=CMD],[CMD=pathname of gpg executable [/usr/local/bin/gpg]]),
 	if test [ x$withval != x ] -a [ -x $withval ]; then
 	  gpgcmd=$withval
 	else
